lib/Doctrine/DBAL/Platforms/DrizzlePlatform.php

```diff
@@ -468,7 +450,7 @@ protected function getDropPrimaryKeySQL($table)
      */
     public function getDateTimeTypeDeclarationSQL(array $fieldDeclaration)
     {
-        if (isset($fieldDeclaration['version']) && $fieldDeclaration['version'] == true) {
+        if (isset($fieldDeclaration['version']) && $fieldDeclaration['version'] === true) {
```

Comment:
This can be considered a BC break. Could be replaced with `if (!empty($fieldDeclaration['version']))`.